cockatrice/src/dlg_settings.cpp

```diff
@@ -246,6 +249,7 @@ void GeneralSettingsPage::retranslateUi()
     highQualityURLLabel.setText(tr("Custom Card Download URL:"));
     highQualityURLLinkLabel.setText(QString("<a href='%1'>%2</a>").arg(LINKING_FAQ_URL).arg(tr("Linking FAQ")));
     clearDownloadedPicsButton.setText(tr("Reset/Clear Downloaded Pictures"));
+    updateNotificationCheckBox.setText(tr("Notify when client updates are available"));
```

Comment:
@Daenyth  Well the required features deny access to the server and the user is notified which is not disable-able.  The client side setting to disable the notification only disables the popup if the client is missing non-required functionality informing the user that there is a possible update.